### ISSUE TITLE
Set an Agent as "uninstall at next connection"

### ISSUE THREAD

**Issue by Tomatoschewps** [Ylianst/MeshCentral#6584]
**Is your feature request related to a problem? Please describe.**
We have some agents on PC who doesn't connect during the day or once every X week so until now we haven't got the chance to uninstall them and even if we delete them on Meshcentral, they got created again next time they got online.

**Describe the solution you'd like**
A possibility to set an offline agent as "Uninstall on next connection"

**Describe alternatives you've considered**
A specific group that if we move an agent inside, it will get uninstall command on next connection

**Additional context**
Nothing more to say

**Comment by Tomatoschewps** [comment]
Hi @vitko-bg,
I didn't know autoRemoveInactiveDevices until now but for what I've seen it doesn't do what I asked in this request.
I need to uninstall automatically agent that doesn't come online often, not remove inactive Devices.

**Comment by tomsik-radek** [comment]
Agreed, this should be an option. Action1 does it this way. If you remove a device from the dashboard it will uninstall the agent on its own as soon as the server can talk to the agent.
